R/preprocessing.R

```diff
@@ -646,7 +646,70 @@ Load10X_Spatial <- function (
   return(object)
 }
 
+#' Load a 10x Genomics Single Cell Experiment into a \code{Seurat} object
+#'
+#' @inheritParams Read10X
+#' @inheritParams SeuratObject::CreateSeuratObject
+#' @param data.dir Directory containing the H5 file specified by \code{filename}
+#' and the image data in a subdirectory called \code{spatial}
+#' @param filename Name of H5 file containing the feature barcode matrix
+#' @param filter.matrix Only keep spots that have been determined to be over
+#' tissue
+#' @param to.upper Converts all feature names to upper case. This can provide an
```

Comment:
(Preemptive-- this parameter doesn't seem to be implemented yet)
In my opinion, it would be better not to suggest that blind uppercasing is ever an appropriate way of finding mouse-human orthologs, even in an exploratory analysis. 
If the user has some good reason to convert all the symbols to uppercase, they can simply use `toupper` and replace the rownames of the object.

Reply:
This option was put in originally by the seurat team. I'll let them address the usefulness here.